.github/workflows/testing.yml

```diff
@@ -13,18 +13,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['8.1', '8.2']
-        drupal-core: ['10.2.x']
+        php-versions: ['8.1', '8.2', '8.3']
+        drupal-core: ['10.2.x', '10.3.x']
         phpstan: ['0']
         include:
           # Extra run to test older supported Drupal 10.1.x.
           - php-versions: '8.1'
             drupal-core: '10.1.x'
             phpstan: '0'
-          # We only need to run PHPStan once on the latest PHP version.
+          # We only need to run PHPStan once on the latest PHP version with next minor and major drupal version.
```

Comment:
not sure we can do that - for now phpstan should run on the latest drupal prod release which is 10.3.